packages/vortex-common/src/msg.rs

```diff
@@ -190,11 +190,9 @@ pub enum QueryMsg {
         end_epoch: i64,
     },
 
-    GetPosition {
-        account: String,
-        price_denom: String,
-        asset_denom: String,
-    },
+    GetPosition(GetPositionQuery),
```

Comment:
Is there a reason to change to type to `GetPosition(query)` vs `GetPosition {struct}` i don't recall if there's a significant difference in rust

Reply:
actually this is slightly better I think because it makes the `match` a bit cleaner in the query entrypoint